NMS-13425: Add configuration example in tuning ActiveMQ (#3513)
OpenNMS/opennms: docs/modules/reference/pages/configuration/tuning-activemq.adoc

echo 'org.opennms.instance.id=MyNMS' > "$\{OPENNMS_HOME}/etc/opennms.properties.d/instance-id.properties"
----

> Update minion's instance id accordingly to match the OpenNMS instance's. The same id instance property should be present in ${MINION_HOME}/etc/custom.system.properties.

Reviewer: Swap to use the product name (Meridian/Horizon) and split sentences on to separate lines.
```suggestion
Update minion's instance ID accordingly to match the {page-component-title} instance.
The same ID instance property should be present in `${MINION_HOME}/etc/custom.system.properties`.
```

> Update minion's instance id accordingly to match the OpenNMS instance's. The same id instance property should be present in ${MINION_HOME}/etc/custom.system.properties.

Reviewer 2: ```suggestion
Update the Minion's instance id accordingly to match the OpenNMS instance's. 
The same id instance property should be present in ${MINION_HOME}/etc/custom.system.properties.
```

[source, sh]
----

> echo 'org.opennms.instance.id=MyNMS' > "$\{MINION_HOME}/etc/custom.system.properties"

Reviewer: Does this need to have two versions to separate the RHEL/Debian versions of the file path?

----

> CAUTION: If you change the instance id setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix:

Reviewer: Maybe update the note to remind the user of the path to file where this change should go.

Author: I put ${MINION_HOME} here on purpose to make it more generic as the path depends on the distribution so that it can appear not clunky. Let me know if you think it's still better to include the reminder!

Reviewer: I was referring to where the XML code goes for the ActiveMQ settings.

Author: Thanks @mmahacek for pointing that out! I've updated it. Hopefully it's clear now.

> CAUTION: If you change the instance id setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix. The broker's configuration is located in `$\{OPENNMS_HOME}/etc/opennms-activemq.xml`:

Reviewer: Format guide: one sentence per line
```suggestion
CAUTION: If you change the instance ID setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix.
The configuration can be modified with the file `${OPENNMS_HOME}/etc/opennms-activemq.xml`:
```

echo 'org.opennms.instance.id=MyNMS' > "$\{OPENNMS_HOME}/etc/opennms.properties.d/instance-id.properties"
----

> Update minion's instance ID accordingly to match the {page-component-title} instance.

Reviewer 2: ```suggestion
Update the Minion's instance ID accordingly to match the {page-component-title} instance.
```
